tests/unit/io-writer.spec.js

```diff
@@ -0,0 +1,156 @@
+import _ from 'lodash'
+import fs from 'fs'
+import test from 'tape'
+import Excel from 'exceljs'
+import tempWrite from 'temp-write'
+import Writer from '../../src/io/writer'
+
+const readXlsx = function readXlsx (filePath) {
+  const values = []
+  const workbook = new Excel.Workbook()
+  return workbook.xlsx.readFile(filePath)
+    .then(() => {
+      const worksheet = workbook.getWorksheet(1)
+      let headerLen = null
+      worksheet.eachRow((row) => {
+        let rowValues = row.values
+        rowValues.shift()
+
+        if (headerLen)
+          // older exceljs reader version trims last value
+          // if it is an empty string, so we will pad row
+          // to header length with empty string
+          rowValues = [
+            ...rowValues,
+            ...Array(headerLen - rowValues.length).fill(''),
+          ]
+        else
+          headerLen = rowValues.length
+
+        rowValues = _.map(rowValues,
+          item => (_.isObject(item) || _.isNil(item) ? '' : item))
+        values.push(rowValues)
+      })
+      return values
+    })
+}
+
+/* eslint-disable no-unused-vars */
+test(`Writer
+  should throw an error when outputFile is not specified`, (t) => {
+  const expectedOutput = 'Error: OutputFile was not specified'
+
+  try {
```

Comment:
Instead of using `try...catch` why not use `[t.throws](https://github.com/substack/tape#tthrowsfn-expected-msg)`.

Should be more cleaner and clearer.